### ISSUE TITLE
Error con.js

### ISSUE THREAD

**Issue by BlackOsint** [marghe943/kibanaChoroplethMap#3]
When kibana optimize this plugin. Log a error that file conf.js not found.

**Comment by marghe943** [comment]
Hi!! 
Did you correctly install the plugin by doing 
`$ ./install.bash path_to_the_elasticsearch_folder_containing_config_folder`
inside the `path_to_kibana_folder/plugins/kibanaChoroplethMap` ?

**Comment by BlackOsint** [comment]
Yes but i dont install elasticdump. Now i have a new problem. Now i have kibanaChoroplethMap
 installed and running, but i can use the coordinate map. I can use it if i uninstalled kibanaChoroplethMap
.

**Comment by marghe943** [comment]
Do you mean that you don't know what should be the coordinates' format for the data in your index? 
By the way, do you have and index called 'World' in your Elasticsearch?  It should have been created during the. /install.bash command (note: to install correctly this plugin you have to install elasticdump)!

**Comment by BlackOsint** [comment]
I installed and configure the kibanaChoroplethMap. I can use this visualization. But I can't use the kibana plugin coordinate map. But if I uninstall kibanaChoroplethMap I can use the kibana plugin coordinates map again.
